client/actions/index.js

```diff
@@ -1,4 +1,26 @@
 import request from 'superagent'
+// import { push } from 'react-router-redux'
+import {hashHistory} from 'react-router'
+
+
+export const addCaption = (caption, postId) => {
+  return dispatch => {
+    const target = `http://localhost:3000/posts/${postId}`
```

Comment:
Let's use `/posts/${postId}` as the value for `target` so we don't run into issues when we deploy.